test/Form_test.js

```diff
@@ -72,15 +69,29 @@ describe("Form", () => {
     const formData = {foo: "invalid"};
 
     function FieldTemplate(props) {
-      const {id, classNames, label, help, required, description, errors, children} = props;
+      const {
+        id,
+        classNames,
+        label,
+        help,
+        rawHelp,
+        required,
+        description,
+        rawDescription,
+        errors,
+        rawErrors,
+        children,
+      } = props;
       return (
         <div className={"my-template " + classNames}>
           <label htmlFor={id}>{label}{required ? "*" : null}</label>
-          <DescriptionField id={`${id}__description`} description={description} />
+          {description}
+          <span id={`${id}__raw_description`}>{`${rawDescription} rendered from the raw format`}</span>
```

Comment:
I think we should keep the `${id}__description` format here, e.g. so users can style both using a same CSS rule.